### PR TITLE
DATA-629: Added Airbyte configuration documentation

### PR DESCRIPTION
**Pull request by paszynski**
`<description>`

Resolves `<issue nr here>`

---
Keep in mind:
- [ ] Documentation updates
- [ ] [Changelog](CHANGELOG.md) updates

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -166,6 +166,59 @@ Data governance configuration
...
 
+Data ingestion configuration
+++++++++++++++++++++++++++++++
+
+Ingestion configuration is divided into two levels:
+
+- General: ``config/<ENV>/ingestion.yml``
+- Ingestion tool related: e.g. ``config/<ENV>/airbyte.yml``
+
+``config/<ENV>/bi.yml`` contains basic configuration of ingestion:
```
Comment (michalrudko) [inline docs/configuration.rst:177]: bi.yml?

File: docs/configuration.rst [hunk 2]
```diff
@@ -166,6 +166,59 @@ Data governance configuration
...
+
+   * - Parameter
+     - Data type
+     - Description
+   * - enable
+     - bool
+     - Flag for enable/disable ingestion option in **dp**.
+   * - engine
+     - enum string
+     - BI tool you want to working with (currently only supported value is ``airbyte``).
```
Comment (michalrudko) [inline docs/configuration.rst:191]: BI tool?

File: docs/configuration.rst [hunk 3]
```diff
@@ -166,6 +166,59 @@ Data governance configuration
...
+     - string
+     - https address of Airbyte deployment that allows to connect to Airbyte API
+   * - connections
+     - array<*connection*>
+     - configurations of Airbyte connection that should be upserted during CI/CD. Documented below
+   * - tasks
+     - array<*task*>
+     - configurations of Airflow tasks used by `dbt-airflow-factory <https://github.com/getindata/dbt-airflow-factory>`_.
+       Allowed *task* options are documented `here <https://dbt-airflow-factory.readthedocs.io/en/latest/configuration.html#id3>`_.
+
```
Comment (michalrudko) [inline docs/configuration.rst:216]: Missing example - the config structure is a bit complex and needs to be supported with an example.

File: docs/configuration.rst [hunk 4]
```diff
@@ -166,6 +166,59 @@ Data governance configuration
...
+
+
+
+
+
 Business Intelligence configuration
 ++++++++++++++++++++++++++++++
```
Comment (michalrudko) [inline docs/configuration.rst:224]: BI config should also be supported with an example

File: docs/configuration.rst [hunk 5]
```diff
@@ -166,6 +166,59 @@ Data governance configuration
...
+   * - airbyte_url
+     - string
+     - https address of Airbyte deployment that allows to connect to Airbyte API
+   * - connections
+     - array<*connection*>
+     - configurations of Airbyte connection that should be upserted during CI/CD. Documented below
+   * - tasks
+     - array<*task*>
+     - configurations of Airflow tasks used by `dbt-airflow-factory <https://github.com/getindata/dbt-airflow-factory>`_.
+       Allowed *task* options are documented `here <https://dbt-airflow-factory.readthedocs.io/en/latest/configuration.html#id3>`_.
```
Comment (michalrudko) [inline docs/configuration.rst:215]: is this configuration up-to-date? https://dbt-airflow-factory.readthedocs.io/en/latest/configuration.html#id3

**Review by michalrudko — CHANGES_REQUESTED** [review]

**Comment by paszynski** [comment]
@michalrudko I updated documentation with some examples as well as divided airbyte.yml into two parts (marked one as not required). Compiled version of configuration.rst file looks like this atm:
image

File: docs/configuration.rst (hunk 3 above)
Comment (paszynski) [reply docs/configuration.rst:214]: ok

File: docs/configuration.rst (hunk 4 above)
Comment (paszynski) [reply docs/configuration.rst:308]: added example

File: docs/configuration.rst (hunk 5 above)
Comment (paszynski) [reply docs/configuration.rst:213]: afaik it is. Even with a typo `asyncronous` that is now widespread

**Review by michalrudko — APPROVED** [review]

**Pull request merged**